.github/workflows/deploy-to-github-pages.yml

```diff
@@ -3,6 +3,8 @@ name: Build and Deploy
 on:
   push:
     branches: [ "main" ]
+
+  workflow_dispatch:
```

Comment:
Is this doing something important?

Reply:
Allows you to do manual deploys from the Actions UX